### DISCUSSION TITLE
Is it not supported to bind Button Command property to a method of ViewModel in Avalonia v11?

### DISCUSSION THREAD

**Discussion by LJN-hzleaper** [AvaloniaUI/Avalonia#12731] [Q&A] [answered]
I bind a button's Command property to a method and find the button is disabled, it seems the binding not works. But  I remember it is supported in v0.10, is this feature removed?

**Comment by maxkatz6** [comment, marked as answer]
> https://docs.avaloniaui.net/docs/next/stay-up-to-date/upgrade-from-0.10#miscellaneousadvanced-scenarios

When binding to methods, the method must either have no parameters or a single object parameter.
As CommandParameter is an object, and automatic casting/mapping was causing problems before.

**Reply by LJN-hzleaper** [reply]
Is the feature "automatic casting/mapping" of command parameter removed?

**Reply by maxkatz6** [reply]
Yes, it was - https://github.com/AvaloniaUI/Avalonia/pull/8905

**Reply by timunie** [reply]
@LJN-hzleaper I prefer to use actual commands whenever possible. Using the MVVM CommunityToolkit, one can simply add [RelayCommand] attribute and will get a command properly implemented.
